src/api/controllers/articleController.js

```diff
@@ -75,6 +77,67 @@ export default class ArticleController {
     });
   }
 
+  /**
+   * This function gets all the articles that are stored in the article table
+   *
+   * @static
+   * @param {Object} req the request
+   * @param {Object} res the response to be sent
+   * @memberof Articles
+   * @returns {Object} res
+   */
+  static async getArticles(req, res) {
+    // default constants
+    const defaultLimit = 10;
+    const defaultOffset = 0;
+    const result = {};
+    const emptyArticleArray = 0;
+
+    // const page = req.query.page || 1;
+    const limit = req.query.limit || defaultLimit;
+    const offset = req.query.offset || defaultOffset;
+    const articles = await Article.findAndCountAll({
```

Comment:
Good work @truestbyheart 

Just something that just occurred to me: let's include the relevant details of the author in the  `getArticles` and `getArticle` methods since we need to display that in the UI.

Reply:
Already implemented it. @MemunaHaruna